modules/docs/docgen/spec/createProgramFromSource.ts

```diff
@@ -0,0 +1,80 @@
+import * as ts from 'typescript';
+import path from 'path';
+
+function getConfig() {
+  const tsconfigPath = ts.findConfigFile('.', ts.sys.fileExists) || '';
+
+  const basePath = path.dirname(tsconfigPath);
+  const {config, error} = ts.readConfigFile(tsconfigPath, ts.sys.readFile);
```

Comment:
Just FYI, `error` and `errors` on the next line aren't used, but fine to keep if desired.